pgactivity/data.py

```diff
@@ -232,11 +233,13 @@ def pg_get_server_information(
             )
             ret = cur.fetchone()
 
+        hr: Optional[Pct] = Pct(0.0)
```

Comment:
If you initialize to 0 here, can the value actually be `None`?

Reply:
When I compute the value if it didn't change since last time it's set to None.

Reply:
It felt weird to set the hit ratio to 100% or 0% in that case.

Reply:
maybe it's worth a comment ?

Reply:
Rather, I'd suggest to rewrite the `hr` assignment below without the ternary `if else`; it'd be clearer.

Reply:
Done